integreat_cms/xliff/utils.py

```diff
@@ -208,7 +208,7 @@ def page_to_xliff(page, target_language, dir_name, only_public=False):
     # Set "currently in translation" status for existing target translation
     if target_page_translation.id:
         target_page_translation.currently_in_translation = True
```

Comment:
```suggestion
    if latest_version := page.get_translation(target_language.slug):
        latest_version.currently_in_translation = True
```
I guess in this case we should only update the status of the latest version (since if e.g. only public versions are exported to XLIFF, the `target_page_translation` will be the latest public version and not the latest version), otherwise a similar problem as before exists :sweat_smile: